Dockerfile-playwright

```diff
@@ -0,0 +1,29 @@
+# Dockerfile for running playwright on CircleCi and locally
+
+# Use the Playwright image as the base image
+FROM mcr.microsoft.com/playwright/python:v1.44.0-jammy
+
+# Create a non-root user and group with a unique UID
+RUN groupadd -r appgroup && useradd -r -u 1010 -g appgroup -m -d /home/app appuser
+
+# Install Python dependencies for testing
+RUN pip install playwright pytest pytest-playwright
```

Comment:
This looks strange. If we have downloaded a playwright image, why do we need to install playwright again?